ui/apps/platform/cypress/tsconfig.json

```
@@ -0,0 +1,9 @@
+{
+    "extends": "../tsconfig.json",
+    "compilerOptions": {
+        // be explicit about types included
+        // to avoid clashing with Jest types
+        "types": ["cypress"]
+    },
+    "include": ["../node_modules/cypress", "./**/*.ts"]
```

I think if we include JS, we'll at least get type hints in all the existing `test.js` files for Cypress built-ins. We won't get actual type checking but it is a start.
```suggestion
    "include": ["../node_modules/cypress",  "./**/*.js", "./**/*.ts"]
```